template-view-duplicates.php

```diff
@@ -29,7 +29,7 @@
                         foreach ( $dt_duplicates as $dt_id => $dt_duplicate ) {
                             if ($dt_duplicate['count'] <= 0) { continue; }
                             echo "<form name='merge_".esc_html( $dt_id )."' method='POST' action='".esc_html( site_url() )."/contacts/".esc_html( $dt_id )."'><input type='hidden' name='dt_contact_nonce' value='".esc_attr( wp_create_nonce() )."'/></form><tr id='".esc_html( $dt_id )."'><td><a>".esc_html( $dt_duplicate['name'] )."</a></td><td>
-                            <a>" . esc_html( $dt_duplicate['count'] ). /* esc_html_e( 'duplicates' ) . */ " duplicates</a></td></tr>";
+                            <a>".esc_html( $dt_duplicate['count'] )." ".esc_html( 'duplicates' )."</a></td></tr>";
```

Comment:
esc_html__( 'duplicates', 'disciple_tools' ) ?

Reply:
Yes - that's all that was needed - change the esc_html_e to esc_html because it was putting the word 'duplicates' before the for each. See any of the current live sites to confirm this issue.

Reply:
Your last commit is just escaping, not translating. Will you add that?

Reply:
Sorry. Missed that. Fixed now. Changes have been committed.